src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
+
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER
+            = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int DAY = 1;
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int WORK_HOURS_INDEX = 2;
+    private static final int PAY_PER_HOUR = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        LocalDate startDate = LocalDate.parse(dateFrom, DATE_FORMATTER).minusDays(DAY);
+        LocalDate endDate = LocalDate.parse(dateTo, DATE_FORMATTER).plusDays(DAY);
+        StringBuilder stringBuilder = new StringBuilder("Report for period " + dateFrom + " - "
+                + dateTo);
```

Comment:
You are using StringBuilder and String Concatenation in one place. 
In case you already have StringBuilder lets use its capabilities.

```
var sb = new StringBuilder();
sb.append("Report ...").append(dateFrom).append(...)
```